src/pages/HomePage.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Page from "../components/Page";
+import { Link, Redirect } from "react-router-dom";
+import { withStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+
+const styles = {
+  //HELP HERE These were the classes body and a before in HomePage.css
+  homepage: {
+    border: "1px solid #eee",
+    textAlign: "center",
+    fontFamily: '"Open Sans", Helvetica, Arial, sans-serif'
+  },
+  link: {
```

Comment:
Oh sorry, just seen #241. I'd argue link colours should be consistent, and set in the theme. (Hi @barto74. ;) But if we _really_ want to make the links a different colour on the home page, then you can set `className={classes.link}` on `<Link>` and this will work.

Reply:
I'll do it as @barto74  intended in the first version. He can decide this.